### PR TITLE
Bisesh added information

### PR DESCRIPTION
**Pull request by Biseshadhikari**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1090"></a>

### CODE REVIEW

File: public/data/bisesh.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Bisesh Adhikari",
+  "type": "personal",
+  "bio": "Student | Tech enthusiast | Aspiring software developer",
+  "avatar": "https://avatars.githubusercontent.com/u/99080674?v=4",
```
Comment (Panquesito7) [inline public/data/bisesh.json:5]: It is recommended to use the profile photo from GitHub using your username directly. That way, it'll be updated when you update your profile photo automatically. 😄
```suggestion
  "avatar": "https://github.com/Biseshadhikari.png",
```

Comment (Biseshadhikari) [reply public/data/bisesh.json:5]: Sorry, I just copied the URL of my profile photo. I will resolve it right away.
Thank you!!

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

File: public/data/bisesh.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Biseshadhikari",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/BiseshAdhikar10",
+      "icon": "Twitter"
+    },
```
Comment (emmalearnscode) [inline public/data/bisesh.json:16]: Please remove the trailing comma on line 16
```suggestion
    }
```

**Review by emmalearnscode — CHANGES_REQUESTED** [review]

File: public/data/bisesh.json (hunk 2 above)
Comment (Biseshadhikari) [reply public/data/bisesh.json]: Thanks for the correction, I made the change.

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**